src/driver/mysql/MysqlDriver.ts

```diff
@@ -640,12 +640,14 @@ export class MysqlDriver implements Driver {
             // console.log("isUnique:", tableColumn.isUnique, this.normalizeIsUnique(columnMetadata));
             // console.log("isGenerated:", tableColumn.isGenerated, columnMetadata.isGenerated);
             // console.log("==========================================");
-
+            const mysqlDefaultLength = 10;
```

Comment:
Not sure if it is always 10(and why), but I experienced this property filled only on int type with even if it wasn't specified in column options.

Reply:
fixes in "sync" area are very dangerous without understanding what and why. First, you compare "width" with "length". Second it might be default to `10`, but usually it depends on column type. So its better to figure those things out and find a proper documentation-based solution.

Reply:
Sorry, I guess I was too tired at the time. Forgot to put [WIP] and named width as length.

Found out why there is 10 - default width declared in `dataTypeDefaults` is decreased by 1 when column is unsigned (it behaves differently with `bigint`). I should be able to fix that tomorrow.